pkg/tilesets/tilesets.go

```diff
@@ -141,3 +141,37 @@ func StandardTileSet() TileSet { //nolint:gocyclo // shallow loops for adding ti
 		Tiles:        tiles,
 	}
 }
+
+func EveryTileOnceTileSet() TileSet { //nolint:gocyclo // shallow loops for adding tiles
+	var tiles = []tiles.Tile{}
+
+	tiles = append(tiles, tiletemplates.MonasteryWithoutRoads()) // 1
+	tiles = append(tiles, tiletemplates.MonasteryWithSingleRoad().Rotate(1))
+	tiles = append(tiles, tiletemplates.StraightRoads()) // 3
```

Comment:
Since there aren't any loops here (like in the standard tileset), this could be simplified as:
```go
var tiles = []tiles.Tile{
    tiletemplates.MonasteryWithoutRoads(),
    tiletemplates.MonasteryWithSingleRoad().Rotate(1),
    tiletemplates.StraightRoads(),
    ...
}
```